### ISSUE TITLE
rtv console (NO X, Linux), mpv does not accept key input running in rtv.

### ISSUE THREAD

**Issue by mikedebian** [michael-lazar/rtv#390]
Hello

When in my tty and I run rtv --enable-media and select a video, gifv or whatever, it plays the selected link using mpv (or mplayer if configured) in a window.

The player will then not register key input like "q" or any other key (to quit the video) under rtv.
Using the player without rtv, it accepts keystrokes.

This makes it hard to quit a video if I change my mind and do not want to see the video after all, or if the video is very long, I have to wait for it to end.

This is not a problem (example) using w3mimgdisplay to show images, as it does accept key input (q) to quit w3mimgdisplay.

**Comment by michael-lazar** [comment]
Thanks for submitting a bug report. I think that this is a limitation in the example mailcap command because it pipes the video from youtube-dl to mpv. E.g. If I type this straight into my terminal then I can't quit either:

 ```youtube-dl -q -o - 'https://www.youtube.com/watch?v=jwC544Z37qo' | mplayer -cache 8192 -vo caca -quiet -```

Do you have a better example of a command that we can use? Do mpv/mplayer support downloading directly from URLs?

**Comment by mikedebian** [comment]
Well, perhaps an easier as mpv supports directly playing of links without piping in terminal.

I changed 

``` 
# Ascii videos
video/x-youtube; youtube-dl -q -o - '%s' | mplayer -cache 8192 -vo caca -quiet -; needsterminal
video/*; wget '%s' -O - | mplayer -cache 8192 -vo caca -quiet -; needsterminal 
```

to

```
# Full motion videos
video/x-youtube; mpv -vo drm -quiet '%s'; needsterminal
video/*; mpv -vo drm -quiet '%s'; needsterminal
```

When running rtv --enable-media and selecting gifv and/or video links, it will play it using the framebuffer under rtv.

Input/shortcuts works as expected.

Please try. I don't know if the videooutput drm will work for everybody.

**Comment by michael-lazar** [comment]
I tried your mpv commands out and they worked for me on Ubuntu if I enabled the framebuffer. I was surprised that mpv was able to parse youtube urls 👍 . I couldn't get it to work on macOS because I couldn't figure out how to install mpv with the drm driver. Even if I did install it, I'm not sure if the framebuffer stuff works on macOS.

I propose that we put your mpv commads first in the example mailcap configuration, with a comment mentioning that they require a framebuffer to run. We can keep the mplayer libcaca commands at the bottom so that they're available for people to play around with.
